dotCMS/src/main/java/com/dotcms/browser/BrowserAPI.java

```diff
@@ -0,0 +1,583 @@
+
+					WorkflowScheme wfScheme = APILocator.getWorkflowAPI().findScheme(action.getSchemeId());
+                    Map<String, Object> wfActionMap = new HashMap<String, Object>();
+                    wfActionMap.put("name", action.getName());
+                    wfActionMap.put("id", action.getId());
+                    wfActionMap.put("icon", action.getIcon());
+                    wfActionMap.put("assignable", action.isAssignable());
+                    wfActionMap.put("commentable", action.isCommentable()
+                            || UtilMethods.isSet(action.getCondition()));
+
+                    final String actionName = Try.of(() ->LanguageUtil.get(user, action.getName())).getOrElse(action.getName());
+                    final String schemeName = Try.of(() ->LanguageUtil.get(user,wfScheme.getName())).getOrElse(wfScheme.getName());
+                    
+                    final String actionNameStr = (showScheme) ? actionName +" ( "+schemeName+" )" : actionName;
+
+                    wfActionMap.put("wfActionNameStr",actionNameStr);
+                    wfActionMap.put("hasPushPublishActionlet", action.hasPushPublishActionlet());
+                    wfActionMapList.add(wfActionMap);
+                }
+
+            }
+
+
+        }
+    }
+    @Deprecated
+    public Map<String, Object> getFolderContent(User user, String folderId,
+                    int offset, int maxResults, String filter, List<String> mimeTypes,
+                    List<String> extensions, boolean showWorking, boolean showArchived, boolean noFolders,
+                    boolean onlyFiles, String sortBy, boolean sortByDesc, boolean excludeLinks, long languageId)
+                    throws DotSecurityException, DotDataException {
+        
+        
+        return getFolderContent( user,  folderId,
+                         offset,  maxResults,  filter,  mimeTypes,
+                        extensions,  showWorking,  showArchived,  noFolders,
+                         onlyFiles,  sortBy,  sortByDesc,  excludeLinks,  languageId, false);
+        
+    }
+    
+	/**
+	 * Gets the Folders, HTMLPages, Links, FileAssets under the specified folderId.
+	 *
+	 * @param user
+	 * @param folderId
+	 * @param offset
+	 * @param maxResults
+	 * @param filter
+	 * @param mimeTypes
+	 * @param extensions
+	 *
+	 * @param showWorking   If true, returns the working version of HTMLPages, Links and FileAssets in the folder.
+	 * 						If false, returns the live version of HTMLPages, Links and FileAssets in the folder.
+	 *
+	 * @param showArchived  If true, includes the archived version of HTMLPages, Links and FileAssets in the folder.
+	 *
+	 * @param noFolders
+	 * @param onlyFiles
+	 * @param sortBy
+	 * @param sortByDesc
+	 * @param excludeLinks
+	 * @return
+	 * @throws DotSecurityException
+	 * @throws DotDataException
+	 */
+    @Deprecated
+	public Map<String, Object> getFolderContent(User user, String folderId,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'user' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5337824)